packages/earth-shared/src/components/user-menu/UserMenu.tsx

```diff
@@ -96,9 +96,18 @@ export const UserMenu = (props: IProps) => {
                   </li>
                 </>
               ) : (
-                <li className="marapp-qa-signin">
-                  <a onClick={compose(onLogin, toggleDrop)}>Sign in</a>
-                </li>
+                <>
+                  <li className="marapp-qa-signin">
+                    <a onClick={compose(onLogin, toggleDrop)}>Sign in</a>
+                  </li>
+                  <li className="marapp-qa-signup">
+                    <a
+                      onClick={compose(onLogin.bind(this, { initialScreen: 'signUp' }), toggleDrop)}
+                    >
```

Comment:
UserMenu component shouldn't know anything about the implementation details of login / sign in / sign up system.
Please create another prop called `onSignUp` and pass it to the component.
```suggestion
                    <a
                      onClick={compose(onSignup, toggleDrop)}
                    >
```